feder/monitorings/templates/monitorings/monitoring_draft_list.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'monitorings/base_monitoring_detail.html' %}
+{% load i18n humanize %}
+{% block content_object %}
+    {% include 'monitorings/_tabs.html' with tab='drafts' %}
+    <h3 class="sr-only">{% trans 'Drafts' %}</h3>
+    {% if object_list %}
+        {% for object in object_list %}
+            {% include 'monitorings/_letter.html' %}
+        {% endfor %}
+        {% if object_list.has_previous or object_list.has_next %}
+            <ul class="pager">
+                {% if object_list.has_previous %}
+                    <li>
+                        <a href="{% url 'monitorings:drafts' slug=object.slug page=object_list.previous_page_number %}">
+                            &laquo; {% trans 'previous' %}
+                        </a>
+                    </li>
+                {% endif %}
+                <li>{% blocktrans with no=object_list.number total=object_list.paginator.num_pages %}Page {{ no }} of
+                    {{ total }}{% endblocktrans %}</li>
+                {% if object_list.has_next %}
+                    <li>
+                        <a href="{% url 'monitorings:drafts' slug=object.slug page=object_list.next_page_number %}">
+                            {% trans 'next' %} &raquo;
+                        </a>
+                    </li>
+                {% endif %}
+            </ul>
+        {% endif %}
+    {% else %}
+        <div class="gray">
+            <p>
+                {% trans 'No rows.' %}
```

Comment:
Wiem, że ja stosuje lakoniczne opisy. Ale może masz ochotę to gramatycznie rozszerzyć?

Reply:
Przychodzi mi do głowy tylko trochę bardziej przyjazne tłumaczenie: "Brak wpisów" zamiast "Brak wierszy"